lib/platform/git/storage.ts

```diff
@@ -347,12 +362,10 @@ class Storage {
           }
         }
         await git.commit(message);
-        await git.push([
-          'origin',
-          `${branchName}:${branchName}`,
-          '--force',
-          '-u',
-        ]);
+        await git.push('origin', `${branchName}:${branchName}`, {
```

Comment:
Should we move these git changes to a smaller, separate PR to merge first?

Reply:
ok, no problem. give me a few minutes 😄

Reply:
done, see #3701

Reply:
Merged